app/main.py

```diff
@@ -1,34 +1,91 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row: int = row
+        self.column: int = column
+        self.is_alive: bool = is_alive
 
 
```

Comment:
The `Deck` class has an attribute `is_alive` which is not used anywhere in your code. According to the checklist, you should remove unused attributes. Please remove it if it is not used.

Reply:
I use is.alive in fire !!!